Source/Private/common.ps1

```diff
@@ -88,7 +96,7 @@ function _callAPI {
       }
 
       # We have to remove any extra parameters not used by Invoke-RestMethod
-      $extra = 'NoProject', 'UseProjectId', 'Area', 'Resource', 'SubDomain', 'Id', 'Version', 'JSON', 'ProjectName', 'Team', 'Url', 'QueryString', 'AdditionalHeaders'
+      $extra = 'NoProject', 'UseProjectId', 'Area', 'Resource', 'SubDomain', 'Id', 'Version', 'JSON', 'ProjectName', 'Team', 'Url', 'QueryString', 'AdditionalHeaders','CustomBearer'

```

Comment:
@DarqueWarrior is this correct to add this parameter to be removed here?

Reply:
Yes. Any parameters that are not used by Invoke-RestMethod need to be removed before we splat them to Invoke-RestMethod.